README.md

```diff
@@ -28,7 +28,14 @@ The automatic installation script works on Debian-based distributions and Arch L
 
 #### Manual
 
-For instructions on manual installations and deployments, see the [wiki page](https://github.com/redpwn/rctf/wiki/Manual-Deployment).
```

Comment:
y remove this? I moved it to the wiki cause most people won't manual install

Reply:
The manual install is short enough to include in the README now I think, no more copying around of config files. It's also useful for installing on Amazon AMIs for example, I think get.rctf doesn't support that right now.